README.md

```diff
@@ -245,11 +245,17 @@
 ### Term representation
 
 * <a name="underscores-in-numerics"></a>
-  Add underscores to large numeric literals to improve their readability.
+  Add underscores to base-10 numeric literals that have six or more digits.
   <sup>[[link](#underscores-in-numerics)]</sup>
 
   ```elixir
+  # Bad
+  num = 1000000
+  num = 1_500
```

Comment:
Hmmm, so you're also saying _don't_ if there _aren't_ at least six?  (Or at least that four is too few, and maybe we can agree that five is borderline?)  If you do mean if and _only_ if there are at least six, I'd suggest inserting an "only" up at line 248 (eg, "Add underscores only to etc.").

If you're willing to entertain five as a borderline case, maybe 248 could be extended to include something like "Do _not_ add them to literals with four or fewer digits."  (Which, by leaving the gap, _implies_ that 5 is a matter of taste.  We _could_ even be absolutely clear and _state_ that.)